basics/file_os_create.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func main() {
+	f, err := os.Create("a.txt")
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	l, err := f.WriteString("Hello World")
+	if err != nil {
+		fmt.Println(err)
+		f.Close()
```

Comment:
IDE is pointing out 'Unhandled error' for f.Close()
Do we need to close() here?

Reply:
fixed it in the new commit.